cmd/mailchain/internal/http/handlers/send_message.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/mailchain/mailchain/crypto/secp256k1"
 	"github.com/mailchain/mailchain/errs"
 	"github.com/mailchain/mailchain/internal/address"
-	"github.com/mailchain/mailchain/internal/envelope"
+	env "github.com/mailchain/mailchain/internal/envelope"
```

Comment:
Mostly I have have seen the variable name altered rather than the import statement.  I'd suggest `env, err := envelope.ParseEnvelope(req.Body.Envelope)` > `envelope, err := env.ParseEnvelope(req.Body.Envelope)` as `envelope` would be more consistent across the codebase. Thoughts?

Reply:
Alright, that makes more sense. I agree that my approach is rather misleading or confusing, I will alter it ASAP.